submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionRequestDocDataRepository.kt

```diff
@@ -312,19 +313,19 @@ class SubmissionRequestDocDataRepository(
             .Builder<Criteria>()
             .apply {
                 add(where(SUB_STATUS).`in`(PROCESSING_STATUS))
-                filter.accNo?.let { add(where("$SUB.$SUB_ACC_NO").`is`(it)) }
-                filter.type?.let { add(where("$SUB.$SUB_SECTION.$SEC_TYPE").`is`(it)) }
-                filter.rTimeFrom?.let { add(where("$SUB.$SUB_RELEASE_TIME").gte(it.toString())) }
-                filter.rTimeTo?.let { add(where("$SUB.$SUB_RELEASE_TIME").lte(it.toString())) }
+                filter.accNo?.let { add(where("$RQT_PROCESSING_INFO.$SUB.$SUB_ACC_NO").`is`(it)) }
+                filter.type?.let { add(where("$RQT_PROCESSING_INFO.$SUB.$SUB_SECTION.$SEC_TYPE").`is`(it)) }
+                filter.rTimeFrom?.let { add(where("$RQT_PROCESSING_INFO.$SUB.$SUB_RELEASE_TIME").gte(it.toString())) }
+                filter.rTimeTo?.let { add(where("$RQT_PROCESSING_INFO.$SUB.$SUB_RELEASE_TIME").lte(it.toString())) }
                 filter.keywords?.let { add(keywordsCriteria(it)) }
-                filter.released?.let { add(where("$SUB.$SUB_RELEASED").`is`(it)) }
+                filter.released?.let { add(where("$RQT_PROCESSING_INFO.$SUB.$SUB_RELEASED").`is`(it)) }
             }.build()
             .toTypedArray()
 
     private fun keywordsCriteria(keywords: String) =
         Criteria().orOperator(
-            where("$SUB.$SUB_TITLE").regex("(?i).*$keywords.*"),
-            where("$SUB.$SUB_SECTION.$SEC_ATTRIBUTES").elemMatch(
+            where("$RQT_PROCESSING_INFO.$SUB.$SUB_TITLE").regex("(?i).*$keywords.*"),
```

Comment:
I suspect many index changes are required given data strucrture has been changed. We may need to change manually archive collection, this one a litle more complicated as some documents has one structure and others will have other.